resources/js/Pages/Praktikum.vue

```diff
@@ -2128,7 +2109,13 @@ export default {
           globe.listAllPraktikan = response.data.all_praktikan;
           globe.praktikanLeft = response.data.all_praktikan.length % response.data.all_asisten.length;
           globe.praktikanComplete = parseInt(response.data.all_praktikan.length / response.data.all_asisten.length);
-          
+          globe.totalGroup = response.data.totalGroup;
```

Comment:
LGTM but me personally still not sure if we re not using the `runmod` mode (the `123` mode)

Reply:
as for runmod 123, I've discuss it with another aslab, and for now it won't be used because of the reason that I have stated